docs/configure/overview.md

```diff
@@ -31,6 +31,10 @@ The `main.js` configuration file is a [preset](../addons/addon-types.md) and as
 - `webpackFinal` - custom [webpack configuration](./webpack.md#extending-storybooks-webpack-config).
 - `babel` - custom [babel configuration](./babel.md).
 
+<div class="aside">
+  Tip/Note: Customize the default story by referencing it first in the `stories` array.
+</div>
```

Comment:
@doublejosh sorry for the misleading here. When i commented out and told you to use the div with "Note/Tip" i was saying that either Note or Tip works for me. 

I'll leave it to you and let me me know when you made the changes so that we can get this through. Sounds good?